src/storage.js

```diff
@@ -23,4 +26,41 @@ module.exports = {
   getFeeds: () => redis.smembers(FEEDS),
 
   getFeed: feedID => redis.hgetall(feedID),
+
+  //  addPost
+  addPost: async post => {
+    const postId = await redis.incr(POST_ID);
+    await redis
+      .multi()
+      .hmset(
+        postId,
```

Comment:
Let's try using `post.guid` here, and not bother with `postId` or `POST_ID` above.  I think the `guid` should be unique enough to use here, but we can experiment.